dependency-check-suppressions.xml

```diff
@@ -32,20 +32,17 @@
     <cve>CVE-2022-23221</cve>
     <cve>CVE-2022-45868</cve>
   </suppress>
-  <suppress until = "2023-09-01">
-    <notes><![CDATA[
-   file name: android-json-0.0.20131108.vaadin1.jar
-   ]]></notes>
+  <suppress until = "2023-10-06">
```

Comment:
could we remove the suppressions that don't seem to be raising an issue anymore

Reply:
Have removed the old suppressions.